modules/two-factor-authentication/backend/modules/two-factor-authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.23 on 2021-06-19 20:54
```

Comment:
As the very last step of this review process, lets squash the migrations into a single one.